helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/collectiontransformer/StringListView.java

```diff
@@ -38,16 +38,30 @@ public String set(int index, String element) {
     return previousValue;
   }
 
+  public String set(int index, Utf8 element) {
```

Comment:
should add this method in base class/interface?

Reply:
Base class is java's AbstractList. We cant modify that